template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Class1.cs

```diff
@@ -1,8 +1,7 @@
-﻿using System;
+﻿namespace Company.ClassLibrary1
+using System;
```

Comment:
I think we should keep the ordering as it was, i.e.:
``` csharp
using System;

namespace Company.ClassLibrary1;

public class Class1
{
}
```

Reply:
is correct syntax `namespace Company.ClassLibrary1;` or `namespace Company.ClassLibrary1` (note semicolon)?

Reply:
My understanding is the semicolon is required: https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/file-scoped-namespaces.md